docs/operations-guide/environment-variables.md

```diff
@@ -1,61 +1,810 @@
+Only available in Enterprise Edition<br>
+Type: string<br>
+Default: `"frontend_client/favicon.ico"`
+
+Path or URL to favicon file.
+
+#### `MB_APPLICATION_LOGO_URL`
+
+Only available in Enterprise Edition<br>
+Type: string<br>
+Default: `"app/assets/img/logo.svg"`
+
+Path or URL to logo file. For best results use SVG format.
+
+#### `MB_APPLICATION_NAME`
+
+Only available in Enterprise Edition<br>
+Type: string<br>
+Default: `"Metabase"`
+
+Replace the word `Metabase` wherever it appears with the defined string.
+
+#### `MB_ASYNC_QUERY_THREAD_POOL_SIZE`
+
+Type: integer<br>
+Default: `50`<br>
+Since: 0.35.0
+
+Maximum number of async Jetty threads. If not set, then [MB_JETTY_MAXTHREADS](#MB_JETTY_MAXTHREADS) will be used, otherwise it will use the default.
+
+#### `MB_BREAKOUT_BIN_WIDTH`
+
+Type: double<br>
+Default: `10.0`
+
+When using the default binning strategy for a field of type Coordinate (such as Latitude and Longitude), this number will be used as the default bin width (in degrees).
+
+#### `MB_BREAKOUT_BINS_NUM`
+
+Type: integer<br>
+Default: `8`
+
+When using the default binning strategy and a number of bins is not provided, this number will be used as the default.
+
+#### `MB_CHECK_FOR_UPDATES`
+
+Type: boolean<br>
+Default: `true`
+
+Identify when new versions of Metabase are available.
+
+#### `MB_CLIENT`
+
+Type: string<br>
+Default: `null`
+
+Used to recognize the Mac App client, which then defaults to `"OSX"`.
+
+#### `MB_COLORIZE_LOGS`
+
+Type: boolean<br>
+Default: `true`
+
+Color log lines. When set to `false` it will disable log line colors. This is disabled on Windows. Related to [MB_EMOJI_IN_LOGS](#MB_EMOJI_IN_LOGS).
+
+#### `MB_CUSTOM_FORMATTING`
+
+Type: string<br>
+Default: `"{}"`
+
+JSON object keyed by type, containing formatting settings.
+
+#### `MB_CUSTOM_GEOJSON`
+
+Type: string<br>
+Default: `"{}"`
+
+JSON object containing information about custom GeoJSON files for use in map visualizations instead of the default US State or World GeoJSON.
+
+#### `MB_DB_AUTOMIGRATE`
+
+Type: boolean<br>
+Default: `true`
+
+When set to `false`, Metabase will print migrations needed to be done in the application database and exit. Those migrations need to be applied manually. When `true`, Metabase will automatically make changes to the application database. This is not related to migrating away from H2.
+
+#### `MB_DB_CONNECTION_TIMEOUT_MS`
+
+Type: integer<br>
+Default: `5000`
+
+Timeout in milliseconds for connecting to the application database.
+
+#### `MB_DB_CONNECTION_URI`
+
+Type: string<br>
+Default: `null`
+
+A JDBC-style connection URI that can be used instead of most of `MB_DB_*` like [MB_DB_HOST](#MB_DB_HOST). Also used when certain Connection String parameters are required for the connection. The connection type requirement is the same as [MB_DB_TYPE](#MB_DB_TYPE).
+
+Example: `postgres://dbuser:dbpassword@db.example.com:port/mydb?ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactorydbpassword@db.example.com`
```

Comment:
The end of this line looks suspicious:
`NonValidatingFactorydbpassword@db.example.com`

Reply:
```suggestion
Example: `postgres://dbuser:dbpassword@db.example.com:port/mydb?ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactory`
```
Thanks. Perhaps bad copy-paste when I was juggling so many variables.